src/crypter.js

```diff
@@ -7,9 +7,11 @@ const toUrlSafeBase64 = (input: Buffer) => input
   .toString('base64')
   .replace(/\+/g, "-")
   .replace(/\//g, "_")
-  .replace(/=*$/, m => m.length.toString());
+  .replace(/=*$/, m => m.length.toString()); // why not just strip everything?
```

Comment:
Because we actually need to know how many equal signs are present to recover. They are converted since equal is not web safe

Reply:
moved to separate file